docs/upgrade.md

```diff
@@ -102,6 +102,14 @@ Replace all instances of `show` and `before:show` with `render` and
 `before:render`. If you want the view to be visible in the DOM, then listen to
 the `dom:refresh` event.
 
+### Changes to `region.show()`
+
+The `region.show()` method (and by extension `showChildView()`) made a change to
```

Comment:
Cheers, I was intending to get that across but I've not communicated that clearly enough. I'll amend it now.